_docs/2.0.0/usage/run.md

```diff
@@ -77,7 +100,7 @@ Steps to configure the scripts are as follows:
 * Copy all the scripts in `scripts` directory to a location in your environment.
 * Copy `enceladus_env.template.sh` to `enceladus_env.sh`.
 * Change `enceladus_env.sh` according to your environment settings.
-* Use `run_standardization.sh` and `run_conformance.sh` scripts instead of directly invoking `spark-submit` to run your jobs.
+* Use `run_standardization.sh`, `run_conformance.sh` and `run_standardization_conformance.sh` scripts instead of directly invoking `spark-submit` to run your jobs.
```

Comment:
```suggestion
* Use `run_standardization.sh` and `run_conformance.sh` or `run_standardization_conformance.sh` scripts instead of directly invoking `spark-submit` to run your jobs.
```